xcm/pallet-xcm/src/lib.rs

```diff
@@ -106,6 +116,13 @@ pub mod pallet {
 			+ GetDispatchInfo
 			+ IsType<<Self as frame_system::Config>::Call>
 			+ Dispatchable<Origin = <Self as Config>::Origin, PostInfo = PostDispatchInfo>;
+
+		/// The maximum number of items that we can store in the version discovery queue.
+		type VersionDiscoveryQueueSize: Get<u32>;
+
+		/// The latest supported version that we advertise. Generally just set it to
+		/// `pallet_xcm::CurrentXcmVersion`.
+		type AdvertizeXcmVersion: Get<XcmVersion>;
```

Comment:
```suggestion
		type AdvertisedXcmVersion: Get<XcmVersion>;
```

maybe doesnt need to be past tense, but should be US english for a public api